README.md

```diff
@@ -1,24 +1,123 @@
-# README
+<!-- PROJECT SHIELDS -->
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
 
-Things you may want to cover:
 
-* Ruby version
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="EPEOLOTRYREPOLINKHERE">
+    <img src="https://user-images.githubusercontent.com/72179421/127752084-8fdb65d3-222d-4daa-a608-4a612440156a.jpg">
+  </a>
 
-* System dependencies
+  <h3 align="center">e . p . e . o . l . a . t . r . y</h3>
 
-* Configuration
+  <p align="center">
+    an app to catalog your literary and vernacular journey. <br> propelling your pronunciation prowess of perplexing prose <br> <em>epi...olive...uh...tree?</em> --> <strong>epeolatry</strong> <br>
+    <br />
+    <a href="https://epeolatrys-rex.herokuapp.com/">View Demo</a>
+    ·
+    <a href="BUGSREPORTLINKHERE">Report Bug</a>
+    ·
+    <a href="FEATUREREQUESTLINKHERE">Request Feature</a>
+  </p>
+</p>
 
-* Database creation
 
-* Database initialization
 
-* How to run the test suite
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents | Compendium of Content</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li><a href="#getting-started">Getting Started</a></li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contribute">Contribute</a></li>
+  </ol>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+<a id="about-the-project">
+<!-- ABOUT THE PROJECT -->
 
-* ...
+## About The Project | <em>project precis</em>
+
+[![Product Name Screen Shot][product-screenshot]](https://example.com)
+
+Reading expands your world. Now you can expand your vocabulary, too - with Epeolatry. Capture the new and interesting words you come across in your literary adventures by looking up and saving words in Epeolatry. Review all of your new words, or words for a particular book, to enrich your vocabulary through reading.
+
+With Epeolatry, you can: 
+- Track the books you’re reading or have read
+- Look up the definition of a word you don’t know while reading a book 
+- Save the word under the book you’re reading
+</a>
+
+<a id="built-with">
+<!-- BUILT WITH -->
+  
+## Built With | <em>constructed using</em>
+
+* [Ruby on Rails | v5.2.6](https://rubyonrails.org/)
+* [Bootstrap | v5.0.1](https://getbootstrap.com)
+</a>
+
+<a id="getting-started">
+<!-- GETTING STARTED -->
+  
+## Getting Started | <em>to commence</em>
+
+Commence Cranium and Concordance Expansion Now... on [HEROKU](https://epeolatrys-rex.herokuapp.com)
+</a>
+
+<a id="usage">
+<!-- USAGE -->
+  
+## Usage | <em>utilization</em>
```

Comment:
Do you think we need this info for the frontend? Or should we just direct folks to Heroku and link to the BE if people want more info?